PowerPointLabs/PowerPointLabs/SyncLab/View/SyncFormatDialog.xaml.cs

```diff
@@ -16,15 +16,26 @@ namespace PowerPointLabs.SyncLab.View
     public partial class SyncFormatDialog : Window
     {
 
+        string originalName;
         FormatTreeNode[] formats = null;
 
-        public SyncFormatDialog(Shape shape) : this(shape, SyncFormatConstants.FormatCategories)
+        public SyncFormatDialog(Shape shape) : this(shape, shape.Name, SyncFormatConstants.FormatCategories)
         {
         }
 
-        public SyncFormatDialog(Shape shape, FormatTreeNode[] formats)
+        public SyncFormatDialog(Shape shape, string formatName, FormatTreeNode[] formats)
         {
             InitializeComponent();
+            formatName = formatName.Trim();
+            if (SyncFormatUtil.IsValidFormatName(formatName))
```

Comment:
This if is used in two places, why not handle it in SyncUtil?
For example changing `bool IsValidFormatName(string name)` to `string GetValidFormatName(string name)`